worker/edge/edge.go

```diff
@@ -110,43 +113,56 @@ type JWTPlaylistClaims struct {
 	Playlist string
 }
 
+func validateToken(w http.ResponseWriter, r *http.Request) bool {
+	token := r.URL.Query().Get("jwt")
+	if token == "" {
+		w.WriteHeader(http.StatusForbidden)
+		_, _ = w.Write([]byte("Forbidden"))
```

Comment:
```suggestion
		http.Error(w, "Missing JWT", http.StatusForbidden)
```
What about using [`http.Error`](https://pkg.go.dev/net/http#Error) for all error responses?